db/migrate/20160711185930_alter_games_add_user_id_column.rb

```diff
@@ -0,0 +1,6 @@
+class AlterGamesAddUserIdColumn < ActiveRecord::Migration
```

Comment:
We likely don't need this relation, as there's already `white_user_id` and `black_user_id` for the two users.